recipes/spscqueue/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(SPSCQueue REQUIRED CONFIG)
+find_package(Threads REQUIRED)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} SPSCQueue::SPSCQueue Threads::Threads)
```

Comment:
It's might not be obvious, but the library itself doesn't include C++11 `thread`, this is why `pthread` is not added to `cpp_info.system_libs` for Linux and FreeBSD.